.github/workflows/release.yaml

```diff
@@ -2,6 +2,7 @@ name: Release operator version, generates a new Docker image and helm charts
 on:
   push:
     tags: ['v*']
+  workflow_dispatch: {}
```

Comment:
Lets not have the manual button for release workflow?

Reply:
I've found this helpful for re-driving a release if there's a transient failure on other projects I maintain. If you don't have the manual trigger, on an error, you'd have to remove the tag and re-release which sends notifications to watchers on the repo.

Reply:
Sounds like we need this in karpenter :)